modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/shibari_worn.dm

```diff
@@ -0,0 +1,330 @@
+				tightness = SHIBARI_TIGHTNESS_LOW
+
+/obj/item/clothing/under/shibari/process(delta_time)
+	var/mob/living/carbon/human/hooman = loc
+	if(tightness == SHIBARI_TIGHTNESS_LOW && hooman.arousal < 15)
+		hooman.adjustArousal(0.6 * delta_time)
+	if(tightness == SHIBARI_TIGHTNESS_MED && hooman.arousal < 25)
+		hooman.adjustArousal(0.6 * delta_time)
+	if(tightness == SHIBARI_TIGHTNESS_HIGH && hooman.arousal < 30)
+		hooman.adjustArousal(0.6 * delta_time)
+
+//stuff to apply processing on equip and add mood event for perverts
+/obj/item/clothing/under/shibari/equipped(mob/user, slot)
+	.=..()
+	var/mob/living/carbon/human/hooman = user
+	if(src == hooman.w_uniform)
+		START_PROCESSING(SSobj, src)
+	if(HAS_TRAIT(hooman, TRAIT_ROPEBUNNY))
+		hooman.apply_status_effect(/datum/status_effect/ropebunny)
+
+//same stuff as above but for dropping item
+/obj/item/clothing/under/shibari/dropped(mob/user, slot)
+	.=..()
+	STOP_PROCESSING(SSobj, src)
+	var/mob/living/carbon/human/hooman = user
+	if(HAS_TRAIT(hooman, TRAIT_ROPEBUNNY))
+		hooman.remove_status_effect(/datum/status_effect/ropebunny)
+
+/obj/item/clothing/under/shibari/torso
+	name = "shibari ropes"
+	desc = "Nice looking rope bondage."
+	icon_state = "shibari_body"
+
+	greyscale_config = /datum/greyscale_config/shibari_clothes/body
+	greyscale_config_worn = /datum/greyscale_config/shibari_worn/body
+	greyscale_config_worn_digi = /datum/greyscale_config/shibari_worn_digi/body
+	greyscale_config_worn_taur_snake = /datum/greyscale_config/shibari_worn_taur_snake/body
+	greyscale_config_worn_taur_paw = /datum/greyscale_config/shibari_worn_taur_paw/body
+	greyscale_config_worn_taur_hoof = /datum/greyscale_config/shibari_worn_taur_hoof/body
+	greyscale_colors = "#bd8fcf"
+
+//processing stuff
+/obj/item/clothing/under/shibari/torso/process(delta_time)
+	. = ..()
+	var/mob/living/carbon/human/hooman = loc
+	if(tightness == SHIBARI_TIGHTNESS_HIGH && hooman.pain < 30)
+		if(hooman.pain < 25)
+			hooman.adjustPain(0.6 * delta_time)
+		if(prob(10))
+			hooman.adjustOxyLoss(5)
+
+/obj/item/clothing/under/shibari/groin
+	name = "crotch rope shibari"
+	desc = "A rope that teases the wearer's genitals."
+	icon_state = "shibari_groin"
+
+	greyscale_config = /datum/greyscale_config/shibari_clothes/groin
+	greyscale_config_worn = /datum/greyscale_config/shibari_worn/groin
+	greyscale_config_worn_digi = /datum/greyscale_config/shibari_worn_digi/groin
+	greyscale_config_worn_taur_snake = /datum/greyscale_config/shibari_worn_taur_snake/groin
+	greyscale_config_worn_taur_paw = /datum/greyscale_config/shibari_worn_taur_paw/groin
+	greyscale_config_worn_taur_hoof = /datum/greyscale_config/shibari_worn_taur_hoof/groin
+	greyscale_colors = "#bd8fcf"
+
+//stuff to apply processing on equip and add mood event for perverts
+/obj/item/clothing/under/shibari/groin/equipped(mob/user, slot)
+	var/mob/living/carbon/human/hooman = user
+	if(hooman?.dna?.mutant_bodyparts["taur"])
+		slowdown = 4
+	else
+		slowdown = 0
```

Comment:
Why are there completely undocumented changes here arbitrarily slowing down people with taur mutant parts on their DNA?

Reply:
Taur groin bondage combines legs bondage in them too